lib/modules/platform/gerrit/types.ts

```diff
@@ -43,21 +43,26 @@ export interface GerritChange {
   labels?: Record<string, GerritLabelInfo>;
   reviewers?: Record<GerritReviewersType, GerritAccountInfo[]>;
   messages?: GerritChangeMessageInfo[];
-  current_revision?: string;
+  current_revision: string;
```

Comment:
Are you sure those new / updated properties are always there?

Reply:
Yes, all the search API used to populate these data is requesting such information through the `o=` query. Therefore, Gerrit API will always provide them.